docs/customize/toc.md

```diff
@@ -62,6 +62,19 @@ Here is a quick overview:
 `url:`
 : An external link to a website (starting with `http` or `https`).
   This will be inserted into your book's Table of Contents, though it will not affect your book's structure (like numbering).
+  When a `title:` entry is provided its text is used instead of the full URL.
+  
+Here is an example to show all three types:
+
+```yaml
+format: jb-book
+root: index
+chapters:
+- file: path/to/chapter1
+- url: https://example.com
+  title: Example website
+- glob: subfolder/other*
+```
```

Comment:
```suggestion
When a `title:` entry is provided its text is used instead of the full URL.

Here is an example to show all three types:

```yaml
format: jb-book
root: index
chapters:
- file: path/to/chapter1
- url: https://example.com
  title: Example website
- glob: subfolder/other*
```
```